.travis.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     - cd ../sample-project
     - mvn clean install
     - cd ..
-    - sh generate_docs.sh
+    - ruby generate_docs.ruby
```

Comment:
any reason for rewriting the shell script in ruby?

Reply:
Easier to traverse and find all the yaml files

Reply:
@flopmeister also has more functionality than a basic shell script.